docs/reference/migration/migrate_7_0/java.asciidoc

```diff
@@ -0,0 +1,9 @@
+[[breaking_70_java_changes]]
```

Comment:
We typically do not add changes here for features adds, preferring to reserve it for changes users need to know to help them migrate; I see this as a feature add. Could you please remove this?